CoC.md

```diff
@@ -114,6 +114,77 @@ following behaviors:
   any other violation of this code. That includes trolling and other forms of
   consistently disruptive behavior.
 
+## Enforcement
+
+We do not tolerate unacceptable behavior from any community member, and there
+are no exceptions for those in positions of power such as maintainers,
+sponsors, funders, or other individuals withe decision making authority.
+Further, people in positions of power can wield it to exacerbate the effects of
+harassment and to diminish the repercussions, those who are informal or formal
+leaders are held to a higher standard.
+
+Anyone asked by another community member to stop unacceptable behavior is
+expected to comply immediately. However, you should not step in on someone
+else's behalf without their consent.
+
+### How to get help
+
+The Community Council is made up established members of the community who
+assist with resolving conflicts within the community.
+
+Currently the Community Council consists of:
+
+- Jen Helsby (`@redshiftzero`) - Lead Engineer - [jen@freedom.press](mailto:jen@freedom.press)
+- Loic Dachary (`@dachary`) - Maintainer - [loic@freedom.press](mailto:loic@freedom.press)
+
+You can contact the whole Community Coucil or members individually.
+
+You should contact the Community Council if you have questions or concerns about
+the Code of Conduct (including improvements) or if you feel that you have
+witnessed a Code of Conduct violation. In the even of a violation either
+directed at yourself or someone else, please contact the Community Council as
+soon as possible through whatever analog or digital medium you are most
+comfortable with.
+
+Should report be about any member of the Community Council or if there is a
+conflict of interest, that member will recuse themselves from the conflict
+resolution proceedings. They will not be involved with the discussion,
+documentation, communications, or decisions made by the rest of the Community
+Council in regards to the incident.
```

Comment:
"in regard to" or "with regard to". See, e.g., https://brians.wsu.edu/2016/05/24/in-regards-to/